spdx/parsers/tagvaluebuilders.py

```diff
@@ -36,7 +36,7 @@ def checksum_from_sha1(value):
     checksum. Returns None if doesn't match CHECKSUM_RE.
     """
     # More constrained regex at lexer level
-    CHECKSUM_RE = re.compile('SHA1:(.+)', re.UNICODE)
+    CHECKSUM_RE = re.compile('SHA1: *(.+)', re.UNICODE)
```

Comment:
Are tabs or other whitespaces also OK there? e.g. would `CHECKSUM_RE = re.compile('SHA1:\s*(.+)', re.UNICODE)` be better or may be it does not make sense? 

And would it make sense to drop trailing or inner whitespaces entirely e.g. may be with `CHECKSUM_RE = re.compile('SHA1:\s*([\S]+)', re.UNICODE)` ?